.github/workflows/actions.yaml

```diff
@@ -0,0 +1,28 @@
+# https://github.com/actions/starter-workflows/blob/main/ci/node.js.yml
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
```

Comment:
We probably only need one node version here.

Reply:
oh good idea. i just enabled 12.x